bots/employee-finder/src/SSW.SophieBot.HttpClientComponents.PersonQuery/EmployeesHelper.cs

```diff
@@ -174,21 +174,9 @@ public static BookingStatus GetBookingStatus(GetEmployeeModel employee, DateTime
             return BookingStatus.Unknown;
         }
 
-        //This function is to
-        // 1. decide if the the working hours are billable hours or internal hours(Client work or internal project)
-        // 2. calculate the billable days according to the billable hours
         public static int GetBilledDays(GetEmployeeModel employee, GetEmployeeProjectModel project, out double billableHours)
         {
-
-
-            if (project != null && project.CustomerName != "SSW")
-            {
-                billableHours = project?.BillableHours ?? 0;
-            }
-            else {
-                billableHours = 0;
-            }
-         
+            billableHours = project?.BillableHours ?? 0;
 
             return billableHours == 0 ? 0 : (int)Math.Ceiling(billableHours / 8);
```

Comment:
I personally do not like the code to be written in this way.
Naming is the best documentation

Reply:
Could you be more specific? But the code can be improved. 

1. Please have a look at this commit to see if it makes sense: https://github.com/SSWConsulting/SSW.SophieBot/pull/641/commits/aff5e0cdc0c7ec2ea48729d63c68a3423d8f0138